pkg/pillar/types/errortime.go

```diff
@@ -12,19 +12,93 @@ import (
 
 // Common error with timestamp
 
+// ErrorSeverity tells the severity type, must be in sync with info api
+type ErrorSeverity int32
+
+const (
+	// ErrorSeverityUnspecified severity unspecified
+	ErrorSeverityUnspecified ErrorSeverity = 0
+	// ErrorSeverityNotice severity notice
+	ErrorSeverityNotice ErrorSeverity = 1
+	// ErrorSeverityWarning severity warning
+	ErrorSeverityWarning ErrorSeverity = 2
+	// ErrorSeverityError severity error
+	ErrorSeverityError ErrorSeverity = 3
+)
+
+//GetErrorSeverity returns ErrorSeverity based on retry count and time spend
+func GetErrorSeverity(retryCount int, timeSpend time.Duration) ErrorSeverity {
```

Comment:
Cool.
Do we want to make the 20,10,2,and 1 as const definitions (such as RetryCountError, RetryTimeError, etc)?

Reply:
Done